tests/test-left_justify.c

```diff
@@ -169,12 +178,19 @@ d128_type d128[] =
 
   {__LINE__, 1E-6143DL, "+1000000000000000000000000000000000e-6176", "%s"},
   {__LINE__, 0.0000000000000001E-6127DL, "+1000000000000000000000000000000000e-6176", "%s"},
-  /* __get_digits_d128 returns +0000000000000000000000000000000001e-6176 for a NaN data. */
-  {__LINE__, 0.0000000000000001E-6140DL, "+0000000000000000000000000000000001e-6176", "%s"},
-  {__LINE__, 0.0000000000000100E-6143DL, "+0000000000000000000000000000000001e-6176", "%s"},
-  {__LINE__, 0.0000000000001000E-6143DL, "+0000000000000000000000000000000001e-6176", "%s"},
-  {__LINE__, 0.0000000000000001E-6143DL, "+0000000000000000000000000000000001e-6176", "%s"},
+
+  {__LINE__, 0.0000000000000001E-6140DL, "+0000000000000100000000000000000000e-6176", "%s"},
+  {__LINE__, 0.0000000000000100E-6143DL, "+0000000000000010000000000000000000e-6176", "%s"},
+  {__LINE__, 0.0000000000001000E-6143DL, "+0000000000000100000000000000000000e-6176", "%s"},
+  {__LINE__, 0.0000000000000001E-6143DL, "+0000000000000000100000000000000000e-6176", "%s"},
```

Comment:
These changes are breaking x86_64.

Reply:
And this seem fixed in the latest patchset.